src/components/Lecturers/Lecturers.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState, useEffect } from 'react';
+import { Container, Wrapper, Button, TitleContainer, Title } from '../../styled/common';
+import { CardsContainer, Card, LecturerImg, LecturerName, LecturerInfo, LecturerImgContainer, LecturerImgColor } from './Lecturers.elements';
+import axios from 'axios';
+export interface Person {
+    id: number;
+    image: string;
+    name: string;
+    title: string;
+    program: string
+}
+
+function Lecturers({}) {
+    const [lecturerData, setLecturerData] = useState([])
+
+    const initialLimit = 8
+    const [limit, setLimit] = useState<number>(initialLimit)
+
+    const showMoreItems = () => {
+        setLimit(limit + 8);
+    };
+
+    useEffect(() => {
+        axios
+            .get('/api/lecturers')
+            .then((res) => {
+                setLecturerData(res.data);
+            })
+            .catch((err) => {
+                console.log(err);
+            })
+    },[]);
+
+
+    let nutriLecturerData = lecturerData.filter( (e: Person)  => e.program == 'nutri')
```

Comment:
V tomto případě je to úplná drobnost, někdy se ale při větších výpočtech bude hodit `useMemo` - https://reactjs.org/docs/hooks-reference.html#usememo

Když se použíje useMemo, budou se lektoři filtrovat pouze při změně vybraných hodnot (obdoba useEffect). Jinak se budou lektoři filtrovat při každém rendrování, což je škoda :)